src/base.js

```diff
@@ -0,0 +1,16 @@
+import Rebase from 're-base';
+import firebase from 'firebase';
+
+const firebaseApp = firebase.initializeApp({
+    apiKey: "AIzaSyDqVXvVWcbWqNGo5Zg16lB1iPzlr0dFEoQ",
```

Comment:
Here, you've exposed the API key in the code directly, which is bad practice and a security vulnerability. Even though this is a private repo, the code could be made public in the future. One of the easiest ways around this (but def not the most secure) is by creating another file called something like `secrets.js` that contains json of all vars that need to be hidden, e.g:

```
module.exports = (() => {
	return {
               firebaseApiKey: 'etc...'
        }
})();
```
Then, you could import the secrets in necessary files with
```
import /*path to secrets*/
/*etc*/
apiKey: secrets.firebaseApiKey
```

Here are some other ways that people integrate secret keys: https://medium.freecodecamp.org/how-to-securely-store-api-keys-4ff3ea19ebda